pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -218,7 +218,7 @@ def _run_binsearch_scaling(trainer, model, new_size, batch_arg_name, max_trials,
         trainer.global_step = 0  # reset after each try
         try:
             # Try fit
-            trainer.fit(model, **fit_kwargs)
+            trainer.tuner._fit(model, **fit_kwargs)
```

Comment:
it's odd we call a private method like this

Reply:
I know... I added the `_` so users don't rely on it, not because I don't want another of our components calling it.

I guess this is the difference between:
- `_foo`: discourage users
- `__foo`: actual private method